lib/delayed/job_groups/job_extensions.rb

```diff
@@ -11,9 +11,7 @@ def ready_to_run(worker_name, max_run_time)
       end
 
       included do
-        if Delayed::JobGroups::Compatibility.mass_assignment_security_enabled?
-          attr_accessible :job_group_id, :blocked
-        end
+        attr_accessible :job_group_id, :blocked if Delayed::JobGroups::Compatibility.mass_assignment_security_enabled?
```

Comment:
Do you know if we still need this code? `attr_accessible` was removed in Rails 4.0 and the [protected_attributes](https://github.com/rails/protected_attributes) gem only supports Rails < 5.0

Reply:
I think we can remove this since we only support rails >= 5.2 now.